/image2rtsp:
   ros__parameters:

      # If camera serves as a source
      camera: False      
      source: "v4l2src device=/dev/video0"

      # If the source is a ros2 topic (default case)
      compressed: True
      topic: "/image_raw/compressed"

      # General setup
      mountpoint: "/back"
      bitrate: "500"
      framerate: "30"
      caps_1: "video/x-raw, framerate ="
      caps_2: "/1,width=640,height=480"
      port: "8554"
      local_only: True     # True = rtsp://127.0.0.1:portAndMountpoint (The stream is accessible only from the local machine)
                           # False = rtsp://0.0.0.0:portAndMountpoint (The stream is accessible from the outside) 
                           # For example, to access the stream running on the machine with IP = 192.168.20.20,
                           # use rtsp://192.186.20.20:portAndMountpoint
